src/js/redux/reducers/covid19/covid19Reducer.js

```diff
@@ -4,13 +4,17 @@
  */
 
 const initialState = {
-    codes: []
+    defCodes: [],
+    overview: {}
 };
 
 const covid19Reducer = (state = initialState, action) => {
     switch (action.type) {
-        case 'SET_CODES': {
-            return Object.assign(state, { codes: action.codes });
+        case 'SET_DEF_CODES': {
+            return Object.assign(state, { defCodes: action.defCodes });
+        }
+        case 'SET_OVERVIEW': {
```

Comment:
`SET_COVID_OVERVIEW`!!! this could trigger on a number of different redux key spaces I would think